toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -60,18 +59,24 @@ class FarmServer;
 
 namespace {
 
+const char *applicationName     = "OpenToonz";
+const char *applicationVersion  = "1.1";
+const char *applicationRevision = "2";
+
 //--------------------------------------------------------------------
 TFilePath getGlobalRoot() {
   TFilePath rootDir;
 
 #ifdef _WIN32
-  TFilePath name(L"SOFTWARE\\OpenToonz\\OpenToonz\\1.0\\FARMROOT");
+  QString regpath = L"SOFTWARE\\OpenToonz\\OpenToonz\\" + applicationVersion + "\\FARMROOT";
```

Comment:
You are adding multiple values of type _const char*_ together here. In C++, this does not concatenate the actual strings, it adds the values of the pointers together, giving you a nonsense result that may crash the application when accessed. I'm really surprised that this code even works on your system.

One of the easiest ways to avoid this is by explicitly constructing a QString from your literal, at least the first one in the chain. As long as one operand is a QString or std::(w)string, the + operator will work since it would then call the corresponding overloaded operator on the QString/std::(w)string class. But if both operands are pointers, you're just adding up the address in memory.